apollo-ios-codegen/Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -1351,6 +1483,7 @@ extension ApolloCodegenConfiguration.OutputOptions {
     deprecatedEnumCases: ApolloCodegenConfiguration.Composition = Default.deprecatedEnumCases,
     schemaDocumentation: ApolloCodegenConfiguration.Composition = Default.schemaDocumentation,
     selectionSetInitializers: ApolloCodegenConfiguration.SelectionSetInitializers = Default.selectionSetInitializers,
+    fieldMerging: ApolloCodegenConfiguration.FieldMerging = Default.fieldMerging,
```

Comment:
What's the reason for adding this parameter to the initializer parameters? Couldn't we just be using `Default.fieldMerging` down in the body of the initializer? That would maintain the signature and functionality both as before and if a user wants the new functionality then they must use the new initializer?